src/main/webapp/app/exercises/shared/participation/participation.component.html

```diff
@@ -82,9 +82,9 @@ <h2>
                             <fa-icon [icon]="controls.iconForSortPropField('repositoryUrl')"></fa-icon>
                         </span>
                     </ng-template>
-                    <ng-template ngx-datatable-cell-template let-value="value">
+                    <ng-template ngx-datatable-cell-template let-value="value" let-row="row">
                         <span *ngIf="value !== null">
-                            <a href="{{ value }}" target="_blank">Repository Link</a>
+                            <a href="{{ adjustRepositoryUrl(row, value) }}" target="_blank">Repository Link</a>
```

Comment:
Not sure how important this is but there's a Codacy warning here (in case that you missed it)

Reply:
thanks, I have added `rel="noreferrer noopener"` to fix the issue